app/controllers/stafftools/groups_controller.rb

```diff
@@ -5,6 +5,15 @@ class GroupsController < StafftoolsController
 
     def show; end
 
+    def destroy
+      grouping = @group.grouping
+
+      @group.destroy
```

Comment:
I think we'd like to handle the case where the group actually didn't get destroyed. Maybe something like?

```ruby
if @group.destroy
  flash[:success] = 'Group was destroyed'
  redirect_to stafftools_grouping_path(grouping.id)
else
  flash[:error] = 'Group was not destroyed'
  render :show
end
```